sdv/constraints/tabular.py

```diff
@@ -519,7 +519,9 @@ def transform(self, table_data):
                 Transformed data.
         """
         table_data = table_data.copy()
-        del table_data[self._column]
+
+        if self._column in table_data:
```

Comment:
I'm wondering if we should (or can) do something more generalized that makes sure any constraint doesn't attempt to access columns not present in the `table_data`. We might not be able to though

Reply:
Maybe we could pass an argument through when know we are transforming on partial table view (e.g. conditions), and only ignore invalid columns in that case. Otherwise we can error out, so it's obvious if the constraint is on a non-present column.

Reply:
> I'm wondering if we should (or can) do something more generalized that makes sure any constraint doesn't attempt to access columns not present in the table_data. We might not be able to though

Well, we actually have that, @amontanez24 ! Those are the `self._constraint_columns` :-)
So in this case, the proper fix would be to add `self._column` to `self._constraint_columns` inside `__init__`, and then change rename `transform` to `_transform`, so the parent `transform` validates that.

On the other side, I would still keep this `if` here, because having the `_transform` be robust enough to not crash if the column is there does not hurt either.